### ISSUE TITLE
Bug: Infinite temperature via decompression

### ISSUE THREAD

**Issue by SunShine214** [petrolpark/Destroy#240]
Using a single pump at 256 rpm, you can reach infinite heat. To reproduce, build a 3x3x3 vat and place the pump pumping out of the vat.  As the mols f air decrease, the pressure stops decreasing, leading to increasing heat. By doing this, the vat will quickly explode due to reaching the number limit for floats.

**Comment by petrolpark** [comment]
Pumping air out usually slightly decreases temperature (room temperature air is removed, cold vacuum remains), so I'm not sure what's causing that. What's your setup?

There is no mechanism for Vats at exploding due to anything to do with temperature. It's likely due to the vacuum being stronger than the pressure differential the Vat can withstand.

**Comment by ToastbrotExe** [comment]
I've also encountered that strange vat behavior, but only in version 0.5 of destroy. I couldn't replicate it in either version 0.6 nor 0.7. I used a 5x5x1 vat made of iron blocks, with 24 super-heated blaze burners below and a pump on top, pumping gas into an open vat. The vat (which was getting all its gases pumped out) reached a temperature of about 1,6584x10^49 °C before exploding.
![2024-01-28_21 14 41](https://github.com/petrolpark/Destroy/assets/159703972/9f4f5533-1c6f-4d51-91cd-85315d9a0598)

**Comment by petrolpark** [comment]
@ToastbrotExe  So this screenshot is in 0.5?

**Comment by ToastbrotExe** [comment]
Yes this screenshot was taken in version 0.5 and I couldn't replicate it in any version higher than that.

**Comment by petrolpark** [comment]
Seems like the pressure changes in 0.6 fixed this then.

**Comment by ToastbrotExe** [comment]
The issue has been recreated in the lastest version of Destroy (0.9) -> #371
